tests/basic/ec/ec-fast-fgetxattr.t is failing

```
ok  18 [     25/     13] <  28> 'touch /mnt/glusterfs/0/file'
tests/basic/ec/ec-fast-fgetxattr.c: In function ‘write_async’:
tests/basic/ec/ec-fast-fgetxattr.c:56:55: error: passing argument 6 of ‘glfs_pwritev_async’ from incompatible pointer type [-Wincompatible-pointer-types]
   56 |     ret = glfs_pwritev_async(glfd, &iov, 1, 0, flags, write_async_cbk, NULL);
      |                                                       ^~~~~~~~~~~~~~~
      |                                                       |
      |                                                       void (*)(glfs_fd_t *, ssize_t,  struct stat *, struct stat *, void *) {aka void (*)(struct glfs_fd *, long int,  struct stat *, struct stat *, void *)}
In file included from tests/basic/ec/ec-fast-fgetxattr.c:9:
/usr/local/include/glusterfs/api/glfs.h:777:57: note: expected ‘glfs_io_cbk’ {aka ‘void (*)(struct glfs_fd *, long int,  struct glfs_stat *, struct glfs_stat *, void *)’} but argument is of type ‘void (*)(glfs_fd_t *, ssize_t,  struct stat *, struct stat *, void *)’ {aka ‘void (*)(struct glfs_fd *, long int,  struct stat *, struct stat *, void *)’}
  777 |                    off_t offset, int flags, glfs_io_cbk fn, void *data) __THROW
      |                                             ~~~~~~~~~~~~^~
not ok  19 [  10100/     32] <  36> 'build_tester tests/basic/ec/ec-fast-fgetxattr.c -lgfapi -Wall -O2' -> ''
tests/basic/ec/../../include.rc: line 348: tests/basic/ec/ec-fast-fgetxattr: No such file or directory
not ok  20 [     34/      1] <  37> 'tests/basic/ec/ec-fast-fgetxattr 192.168.122.213 patchy /file' -> ''
Failed 2/20 subtests
```

--- a/tests/basic/ec/ec-fast-fgetxattr.c
+++ b/tests/basic/ec/ec-fast-fgetxattr.c
@@ -30,8 +30,8 @@ fill_iov(struct iovec *iov, char fillchar, int count)
 }
 
 void
-write_async_cbk(glfs_fd_t *fd, ssize_t ret, struct stat *prestat,
-                struct stat *poststat, void *cookie)
+write_async_cbk(glfs_fd_t *fd, ssize_t ret, struct glfs_stat *prestat,
+                struct glfs_stat *poststat, void *cookie)
 {
     if (ret < 0) {
         fprintf(stderr, "glfs_write failed");

